packages/api-explorer/src/components/SelectorContainer/SdkLanguageSelector.spec.tsx

```diff
@@ -64,7 +64,7 @@ describe('SdkLanguageSelector', () => {
 
   test('it lists all available languages and "All" as options', async () => {
     renderWithRouterAndReduxProvider(<SdkLanguageSelector />, undefined, store)
-    userEvent.click(screen.getByRole('textbox'))
+    await userEvent.click(screen.getByRole('textbox'))
     await waitFor(() => {
```

Comment:
Are the waitFor() calls still necessary if you're awaiting the events? We should probably also `await click()` outside of the waitFor for reasons mentioned in my previous comment.

Reply:
I've removed the `waitFor` statements I could in #1418